operations/maintainer.go

```diff
@@ -29,27 +29,32 @@ import (
 // Maintainer ensures that operations old enough are deleted
 // and that no orphan operations are left in the DB due to crashes/restarts of SM
 type Maintainer struct {
-	smCtx               context.Context
-	repository          storage.Repository
-	jobTimeout          time.Duration
-	markOrphansInterval time.Duration
-	cleanupInterval     time.Duration
+	smCtx                   context.Context
+	repository              storage.Repository
+	jobTimeout              time.Duration
+	markOrphansInterval     time.Duration
+	cleanupInterval         time.Duration
+	operationExpirationTime time.Duration
 }
 
 // NewMaintainer constructs a Maintainer
 func NewMaintainer(smCtx context.Context, repository storage.Repository, options *Settings) *Maintainer {
 	return &Maintainer{
-		smCtx:               smCtx,
-		repository:          repository,
-		jobTimeout:          options.JobTimeout,
-		markOrphansInterval: options.MarkOrphansInterval,
-		cleanupInterval:     options.CleanupInterval,
+		smCtx:                   smCtx,
+		repository:              repository,
+		jobTimeout:              options.JobTimeout,
+		markOrphansInterval:     options.MarkOrphansInterval,
+		cleanupInterval:         options.CleanupInterval,
+		operationExpirationTime: options.ExpirationTime,
 	}
 }
 
 // Run starts the two recurring jobs responsible for cleaning up operations which are too old
 // and deleting orphan operations
 func (om *Maintainer) Run() {
+	om.cleanUpOldOperations()
+	om.markOrphanOperationsFailed()
```

Comment:
@georgifarashev , @pankrator , @KirilKabakchiev 
Any idea why we trigger these immediately on SM start up and not wait until the first ticker to trigger them?

Reply:
because if SM restarts often enough and the initial interval never passes, we theoretically might never do a cleanup